src/hotspot/share/opto/buildOopMap.cpp

```diff
@@ -466,8 +466,12 @@ static void do_liveness(PhaseRegAlloc* regalloc, PhaseCFG* cfg, Block_List* work
           assert(def != 0, "input edge required");
           int first = regalloc->get_reg_first(def);
           int second = regalloc->get_reg_second(def);
-          if( OptoReg::is_valid(first) ) set_live_bit(tmp_live,first);
-          if( OptoReg::is_valid(second) ) set_live_bit(tmp_live,second);
+          if ( def->is_Mach() && def->as_Mach()->get_removed() ) {
+            //do nothing. peephole has removed the node.
+          } else {
+            if( OptoReg::is_valid(first) ) set_live_bit(tmp_live,first);
+            if( OptoReg::is_valid(second) ) set_live_bit(tmp_live,second);
+          }
```

Comment:
Code style. Use next pattern in code you modified (body on separate line and no spaces before/after condition):
```
if (cond) {
  foo;
}
``` 
We want to correct style in all places eventually as we work on code.

Reply:
Done